internal/api/tests/achievement_handler_test.go

```diff
@@ -100,3 +103,102 @@ func TestCreateAchievementHandler(t *testing.T) {
 		})
 	}
 }
+
+var (
+	profileID  = "1"
+	profileID0 = ""
+)
+
+func TestGetAchievementsHandler(t *testing.T) {
+	achSvc := new(mocks.Service)
+	getAchievementHandler := handler.GetAchievementsHandler(context.Background(), achSvc)
+
+	tests := []struct {
+		name               string
+		queryParams        string
+		MockSvc            func(mockSvc *mocks.Service)
+		expectedStatusCode int
+	}{
+		{
+			name:        "Success for fetching achievements",
```

Comment:
Success for fetching achievements -> success_achievement

Reply:
Done